99-make_reports.py

```diff
@@ -53,8 +55,17 @@ def run_report(subject, session=None):
                      subjects_dir=subjects_dir)
     rep.parse_folder(fpath_deriv, verbose=True)
 
-    evokeds = mne.read_evokeds(fname_ave)
+    # Visualize events.
+    raw_filt = mne.io.read_raw_fif(fname=fname_raw_filt,
+                                   allow_maxshield=config.allow_maxshield)
+    events = mne.find_events(raw=raw_filt,
+                             min_duration=config.min_event_duration)
```

Comment:
don't use find_events but use the events from bids file.

Reply:
That's in fact another issue I had been planning to bring up in the future, but since you're mentioning it now…

Currently the pipeline writes all derivative data via vanilla MNE functions, and doesn't use `mne-bids`. This should be changed, IMHO – and you seem to agree. I can amend this PR to do this (for all pipeline results), or open a new PR for this – what do you think?

Reply:
(mind you, the events I process here are those from the filtered raw data, i.e., after the 2nd step of the pipeline has been run)